test/fprime/util/commands_unit_test.py

```diff
@@ -0,0 +1,53 @@
+"""
+(test) fprime-utils commands:
+
+Tests the F prime util commands.
+@author 0x48piraj
+"""
+
+import unittest
+
+
+class CommandsTestCases(unittest.TestCase):
```

Comment:
What are there a bunch of empty tests here?

Reply:
As commented above, it's a skeleton structure and I was thinking of writing tests separately in another pull, just making a placeholder class.

Reply:
@0x48piraj if you start looking into adding test cases, we usually use `pytest` over `unittest`